addons/web/static/src/js/chrome/apps_menu.js

```diff
@@ -41,8 +41,10 @@ var AppsMenu = Widget.extend({
      * Open the first app in the list of apps
      */
     openFirstApp: function () {
-        var firstApp = this._apps[0];
-        this._openApp(firstApp);
+        if (this._apps.length) {
```

Comment:
I prefer preemptive return in JS:
```js
if (!this._apps.length) {
   return;
}
```
Update docstring to tell this is ignored if there are no app.
Maybe add `console.warn()`? To your liking!

Reply:
Euhhh I'd vote against a warn. It's desired that it doesn't try to open any app as there is none. Just my five cents!

Reply:
Another fix, better but longer diff, would be to never call this function if there are no apps in the 1st place.